style.css

```diff
@@ -1,24 +1,36 @@
 
-
 body { 
-       margin: 0px;
-       background-color: #ffffff;
-       font-family: tahoma, sans-serif;
+      margin: 0px;
+      background-color: #ffffff;
+      font-family: tahoma, sans-serif;
     }
 
- 
- 
 #top  {
-	
-     background-color :  #2d2578;
-	 color:white;
-	 padding: 20px 270px 19px;
-	 border: 0px;
-	 margin: 0px;
-	 
-	 
+	  background-color :  #2d2578;
+	  color:white;
+	  padding: 3px 270px 15px;
+	  border: 0px;
+	  margin: 0px;
 	 }
 
+#top ul {
+        margin: 0px 0px 0px 0px;
+        padding: 0px 0px 0px 0px;
+        list-style-type: none;
+       }
 
-
-
+#top ul li {
+	     display: inline;
+         margin: 0px 15px 0px 0px;
```

Comment:
The entire file is not well formatted, which makes it hard to read and follow. You can find formatter plugin which will solve this problem automatically. 
But now you can just manually whrite code in line:
>  +#top ul li {
 +	     display: inline;
 +         margin: 0px 15px 0px 0px;
            .............
            }